packages/pixeloven/cli/src/commands/compile.test.ts

```diff
@@ -50,7 +50,7 @@ describe("@pixeloven/cli", () => {
                 Stub.system.spawn.resolves({
                     status: 0,
                 });
-                const context = await cli.run("compile ts ./some/file.ts");
+                const context = await cli.run("compile tsx ./some/file.ts");
```

Comment:
but it's a ts file?

Reply:
They are just aliases for each other. This was just to get full coverage without needing to write an entire new test to cover one extra branch.